wordpress-mobile/WordPress-FluxC-Android
# [Coupons] Create coupons

Closes #2231.

**To test:**
1. Go to Woo -> Products
2. Click on Fetch products
3. Click on Fetch product categories
4. Navigate back to Woo -> Coupons
5. Select a site
6. Click on Create coupon
7. Tap on the check mark button
8. **Verify that an error is displayed about missing discount type**
9. Select a discount type
10. Tap on the check mark button
11. **Verify that an error is displayed about the missing coupon code**
12. Fill out the coupon code & other properties
13. **Tap on the check mark button and verify that the coupon was created successfully**
14. Go back to coupon buttons screen
15. Tap on the Fetch coupons button
16. **Verify the new coupon is fetched**
